COM_dev:
 /dev/ttyUSB_FTDI_SERVO

